src/main/webapp/index.html

```diff
@@ -43,17 +43,30 @@
     <title>smallcity</title>
   </head>
   <body>
+    <div id="entry-container"></div>
+    <div id="options-container">
+      <button class="geolocate-button" onclick="getGeolocation()"><i class="fas fa-location-arrow"></i> locate me</button>
+      <div class="zipcode-container">
+        your zipcode:
+        <input type="text" id="entryZipCode" name="entryZipCode" value="">
+        <button type="submit" onclick="getZipCode()" class="zipCode-submit-btn"><i class="fas fa-search"></i></button>
+      </div>
+    </div>
+
+    
     <div id="loader-circle-container">
       <div id="loader-circle"></div>
     </div>
```

Comment:
I believe here we pulled out loader-circle and put it before loader-circle-container to make it so the loader wasn't transparent. If you want to make this change you might need to rename loader-circle-container to something different.

This move can also be done for the popup I think

Reply:
You're totally right - just fixed it